scout/server/blueprints/cases/templates/cases/causatives.html

```diff
@@ -22,121 +22,104 @@
   <a class="nav-link" href="{{ url_for('cases.clinvar_submissions', institute_id=institute._id) }}">Clinvar submissions</a>
 </li>
 {% endblock %}
-
 {% block content_main %}
-<div class="container-float">
-  <div class="card panel-default">
-    <div class="card-body">
-      <form action="{{ url_for('cases.causatives', institute_id=institute._id) }}">
-        <div class="row">
-          <div class="col-md-4 col-xs-4">
-            <div class="input-group">
-              <span class="input-group-addon">
-                <span class="glyphicon glyphicon-search"></span>
-              </span>
-              <input name="query" class="typeahead form-control" data-provide="typeahead" autocomplete="off" placeholder="Search gene">
-            </div>
-          </div>
-          <div class="col-xs-4">
-            <button class="btn btn-default form-control">Search</button>
-          </div>
-        </div>
-      </form>
-    </div>
-  </div>
-</div>
-<div class="list-group" style="margin-top: 25px">
-  {% for _, group in variant_groups.items() %}
-  {% set variant = group[0][1] %}
-  <li class="list-group-item">
-    <div class="d-flex justify-content-between">
-      <h5 class="mb-1">{{ variant.hgnc_symbols|join(', ') }}</h5>
-      <span class="text-muted" style="display: inline-block; margin-left: 10px;">
-          {{ variant.chromosome }}:{{ variant.position }} {% if variant.is_par %}
-          <span class="badge badge-info">PAR</span>
-          {% endif %}
-          {{ variant.reference|truncate(30, True) }} → {{ variant.alternative|truncate(30, True) }}
-        </span>
-    </div>
-    <ul class="list-group ist-group-flush">
+
+<br>
```

Comment:
Instead of <br>, to put some space between the cars

Reply:
Thanks for the advise!